crates/command_palette/src/command_palette.rs

```diff
@@ -437,30 +437,6 @@ impl PickerDelegate for CommandPaletteDelegate {
     }
 }
 
-pub fn humanize_action_name(name: &str) -> String {
```

Comment:
Had to move this to avoid circular dependency, not sure if `gpui` is the right destination.